google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/CloudSdkService.java

```diff
@@ -39,6 +40,8 @@ public static CloudSdkService getInstance() {
 
   public abstract void setSdkHomePath(String path);
 
+  public abstract Set<CloudSdkValidationResult> validateCloudSdk(@NotNull Path pathToCloudSdk);
```

Comment:
Fair point, though each consumer of the common lib might want to require (or recommend) different versions so we would want to allow that flexibility. It would be cool to share this logic with the common lib nonetheless.